### PR TITLE
Replace handmade dropdowns with bootstrap dropdowns

### LINKED ISSUE #81: Replace custom Dropdown component with Bootstrap dropdowns and regular <select>s where relevant

### PR DESCRIPTION
**Pull request by eatonphil**
Basic support for dropdowns. Not pretty yet but position-wise they are much nicer. And they are JS-based (using click events) rather than CSS-based (using hover). Closes #81

screen shot 2016-06-15 at 4 47 36 pm
screen shot 2016-06-15 at 4 47 46 pm
screen shot 2016-06-15 at 4 48 06 pm

### CODE REVIEW

**Comment by eatonphil** [comment]
That dropdown was oriented incorrectly.
screen shot 2016-06-15 at 4 54 00 pm

**Comment by eatonphil** [comment]
Actually I'd like to get the colors sorted out in #140 and fix the button colors here afterward.

**Comment by eatonphil** [comment]
Tests look good. Ready for merge.

File: src/components/Dropdown.js [hunk 1]
```diff
@@ -1,36 +1,73 @@
...
+
+export default class Dropdown extends Component {
+  constructor() {
+    super();
+    this.open = this.open.bind(this);
+    this.close = this.close.bind(this);
+  }
+
+  open(evt) {
+    const event = evt || window.event;
```
Comment (SirCmpwn) [inline src/components/Dropdown.js]: This is not necessary

File: src/components/Dropdown.js [hunk 2]
```diff
@@ -1,36 +1,73 @@
...
+    const target = event.currentTarget || event.srcElement;
```
Comment (SirCmpwn) [inline src/components/Dropdown.js]: Neither is this, use `e.target`.

File: src/components/Dropdown.js [hunk 3]
```diff
@@ -1,36 +1,73 @@
...
+    target.parentElement.classList.toggle('open');
+    return false;
+  }
```
Comment (SirCmpwn) [inline src/components/Dropdown.js]: The pattern we (and the broader react world) uses looks something like this:

    open(e) {
        e.target.parentElement.classList.toggle('open');
    }

Shouldn't be necessary to return false either. However, I'd use `this.setState` and manage this in a stateful react fashion over manipulating the DOM directly.

Comment (SirCmpwn) [inline src/components/Dropdown.js]: I think for managing the state of isolated UI components like this we don't need to plug them into the Redux store, we can just use plain React.js state management.

**Pull request merged**